storage3/_async/client.py

```diff
@@ -39,7 +42,13 @@ def _create_session(
         timeout: int,
         verify: bool = True,
         proxy: Optional[str] = None,
+        http_client: Optional[AsyncClient] = None,
     ) -> AsyncClient:
+        if http_client is not None:
+            http_client.base_url = base_url
+            http_client.headers = headers
```

Comment:
Should also apply the other params? Verify, ...

Reply:
I didn't pass those as I don't want them to be override the options for `httpx` client the developer will pass. The only options we need to maintain are the `base_url` and `headers`. The others should be manually configured by the developer if they are passing their own `httpx` client.